sites/website/src/docs/resources/cheat-sheet.md

```diff
@@ -0,0 +1,705 @@
+```
+
+---
+### [@microsoft/fast-foundation](https://www.fast.design/docs/introduction)
+
+**A library that provides foundational building blocks that can be assembled to create new design systems and component libraries.**
+
+* Exports of this package can generally be thought of as un-styled base components that implement semantic and accessible markup and behavior.   
+* Use this library when you want to implement something like Google's Material Design or Twitter Bootstrap.
+
+```ts
+import {
+  Button as FoundationButton,
+  buttonTemplate as template,
+} from "@microsoft/fast-foundation";
+import { css } from "@microsoft/fast-element";
+
+export class MyButton extends FoundationButton {
+  ...
+}
+
+export const buttonStyles = css`
+  :host {
+    background-color: azure;
+  }
+`;
+
+export const myButton = MyButton.compose({
+  baseName: "button",
+  baseClass: FoundationButton,
+  template,
+  styles: buttonStyles,
+  shadowOptions: {
+    delegatesFocus: true,
+  },
+});
+```
+---
+
+### [@microsoft/fast-components](https://www.fast.design/docs/components/getting-started)     
+
+**A component library that implements Microsoft's [FAST Frame Design System](https://www.fast.design/docs/design-systems/fast-frame/).**
+
+* Assembles the building blocks of `@microsoft/fast-foundation` to create its component set. 
+
+* Use this library when you want to integrate [FAST Components](#using-components) into an existing site or app.
+
+To install the components, use either `npm` or `yarn`:
+```shell
+npm install --save @microsoft/fast-components
+``` 
+```shell
+yarn add @microsoft/fast-components
+```
+
+To use a Web Component as a custom element in HTML, the custom element must be registered:
+
+```ts
+import { 
+    fastButton, 
+    provideFASTDesignSystem 
+} from "@microsoft/fast-components";
+
+provideFASTDesignSystem()
+    .register(
+        fastButton() // custom element
+    );
+```
+
+---
+### [@fluentui/web-components](https://www.fast.design/docs/components/getting-started)
+
+**A component library that implements Microsoft's [Fluent Design System](https://www.microsoft.com/design/fluent/#/).**
+
+* Assembles the building blocks of `@microsoft/fast-foundation` to create its component set. 
+
+* Use this library when you want components to look and feel like those found in Microsoft products.
+
+To install the components, use either `npm` or `yarn`:
+```shell
+npm install --save @fluentui/web-components
+``` 
+```shell
+yarn add @fluentui/web-components
+```
+
+To use a Web Component as a custom element in HTML, the custom element must be registered:
+
+```ts
+import { 
+    fluentButton, 
+    provideFluentDesignSystem 
+} from "@fluentui/web-components";
+
+provideFluentDesignSystem()
+    .register(
+        fluentButton() // custom element
+    );
+```
+
+---
+If you are attempting to configure the components for integration into a specific Microsoft product, the following table provides `AccentBaseColor` values you can use:
```

Comment:
This should link to the "Configuring a design token" section and/or provide an example like:
`accentPalette.withDefault(PaletteRGB.create(SwatchRGB.from(parseColorHexRGB(hexColor)!)));`

"AccentBaseColor" isn't in code yet. There is a PR adding support for that, but there are some open design questions we need to resolve.

Reply:
@bheston Thank you - Fluent content has been removed from the cheat sheet per @chrisdholt 's request.